mimic.py

```diff
@@ -46,46 +46,70 @@
 import random
 import sys
 
-__author__ = "???"
+__author__ = "Detrich"
 
 
 def create_mimic_dict(filename):
-    """Returns mimic dict mapping each word to list of words which follow it. 
-    For example:
-        Input: "I am a software developer, and I don't care who knows"
-        Output: 
-            {
-                "" : ["I"],
-                "I" : ["am", "don't"], 
-                "am": ["a"], 
-                "a": ["software"],
-                "software" : ["developer,"],
-                "developer," : ["and"],
-                "and" : ["I"],
-                "I" : ["don't"],
-                "don't" : ["care"],
-                "care" : ["who"],
-                "who" : ["knows"]
-            }
-    """
+    Mimic = {}
```

Comment:
I did not until now notice that every time we want to run print_mimic, we must also run create_mimic_dict. Nice thinking.